.github/workflows/managed-configuration.yaml

```
@@ -0,0 +1,112 @@
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  managed-configuration:
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    name: Managed configuration tests - ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: ${{ matrix.timeout }}
+    continue-on-error: ${{ matrix.optional == true }}
```

actionlint (1.7.11)

[error] property "optional" is not defined in object type {os: string; timeout: number}

(expression)